### PR TITLE
#2799 [Manager] Fix unstable E2E tests

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2799 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/cypress/api/connectorApi.test.js [hunk 1]
```diff
@@ -71,7 +71,9 @@ const setup = () => {
     'connector.name': connectorName,
     name: connectorName,
     topicKeys: [{ group: topicGroup, name: topicName }],
-    workerClusterName,
+    workerClusterKey: {
+      name: workerClusterName,
```
Comment (konekoya) [inline ohara-manager/client/cypress/api/connectorApi.test.js]: So the `group` can be omitted?

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Plugins/usePlugin.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ const usePlugin = () => {
     let stopConnectorsSuccess = true;
 
     const stopConnectorCheckFn = res => {
-      return isUndefined(get(res, 'data.result.state', undefined));
+      return isUndefined(get(res, 'data.result.status.state', undefined));
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Plugins/usePlugin.js]: Hmm, the state is used throughout our codebase. Could you also fix them? Like in API tests :)

File: ohara-manager/client/cypress/api/connectorApi.test.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/cypress/api/connectorApi.test.js]: group is option, we use cluster group is default, so we can give group

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Plugins/usePlugin.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Plugins/usePlugin.js]: This status is only in connector, we only in here use this.

**Pull request merged**